db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Reconfigure/Email.class/__methods__/vmreconfigure_request_approved.rb

```diff
@@ -0,0 +1,45 @@
+#
+# Description: This method is used to email the requester that
+# VM Reconfiguration request has been approved
+#
+# Events: vmreconfigure_request_approved
+# Model Notes:
+# 1. to_email_address - used to specify an email address in the case where the
+#    requester does not have a valid email address. To specify more than one email
+#    address separate email address with commas. (I.e. admin@example.com,user@example.com)
+# 2. from_email_address - used to specify an email address in the event the
+#    requester replies to the email
+# 3. signature - used to stamp the email with a custom signature
+#
+
+# Get miq_request object
+miq_request = $evm.root["miq_request"]
+
+# Build email to requester with reason
+$evm.log('info', "Requester email logic starting")
+
+# Get requester email
+requester = $evm.root['miq_request'].requester
+
+# Get to_email_address from requester.email then from model if nil
+to = requester.email || $evm.object['to_email_address']
+
+# Get from_email_address from model unless specified below
+from = nil
+from ||= $evm.object['from_email_address']
```

Comment:
@billfitzgerald0120 
Since from is being used for the first time in line 28 and line 29. You should be able to just use
from = $evm.object['from_email_address']